javascript/elements/updates_for_element.js

```diff
@@ -152,24 +174,21 @@ export default class UpdatesForElement extends SubscribingElement {
     )
   }
 
-  get query () {
-    return `updates-for[identifier="${this.identifier}"]`
+  processInnerUpdates (block) {
```

Comment:
I'd probably rename this to something that better indicates it's a predicate (right now it sounds like a command), e.g. `processesInnerUpdates` or other way round `ignoresInnerUpdates`. The latter would be better aligned with the attribute `ignore-inner-updates`, fwiw.

Reply:
Sounds good to me. I changed it to simplify the logic (every negative makes comprehension one step harder) but I actually wasn't even aware that there was an `ignore-inner-updates` attribute. Makes good sense, indeed, now that I know.